c_src/sqlite3_nif.c

```diff
@@ -341,6 +341,10 @@ exqlite_changes(ErlNifEnv* env, int argc, const ERL_NIF_TERM argv[])
         return make_error_tuple(env, "invalid_connection");
     }
 
+    if (conn->db == NULL) {
+        return make_error_tuple(env, "connection closed");
```

Comment:
```suggestion
        return make_error_tuple(env, "connection_closed");
```

If you could also make this change to line 389 as well. That string gets turned into an atom. An oversight on my part.